tests/push/test_presentable_names.py

```diff
@@ -51,13 +51,15 @@ def __init__(self, events: Iterable[Tuple[StateKey, dict]]):
             )
 
     async def get_event(
-        self, event_id: StateKey, allow_none: bool = False
```

Comment:
Aside: I'd've guessed that `StateKey` was an alias for `str`, not `Tuple[str, str]`.